### DISCUSSION TITLE
How to Display a BottomSheet Above the NavigationPage AppBar?

### DISCUSSION THREAD

**Discussion by xDaijobu** [syncfusion/maui-toolkit#61] [General]
Hi everyone,

I'm currently working on a project and facing a challenge with implementing a BottomSheet. Specifically, I want the BottomSheet to appear above the NavigationPage AppBar, rather than below it.

I've tried some standard implementations, but the BottomSheet always seems to be constrained by the NavigationPage layout. Is there a known way or workaround to achieve this kind of behavior?

![image](https://github.com/user-attachments/assets/3a4ee26b-4a8f-448c-ad6e-2239aa25b1e2)

Any advice, tips, or examples would be greatly appreciated. Thank you in advance!

Best regards,
Cris

**Comment by PaulAndersonS** [comment]
Hi @xDaijobu ,

We don't have that support with BottomSheet as it is constrained to the Content area. But we plan to provide this support in future releases.

**Comment by xDaijobu** [comment]
Hi @PaulAndersonS,

I see, thanks! Looking forward to the future releases!
